helm/kadalu/charts/operator/templates/rbac.yaml

```diff
@@ -0,0 +1,207 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: kadalu-operator
+rules:
+  - apiGroups:
+      - storage.k8s.io
+      - csi.storage.k8s.io
+    resources:
+      - csidrivers
+      - storageclasses
+    verbs:
+      - create
+      - delete
+      - get
+      - list
+      - update
+  - apiGroups:
+      - kadalu-operator.storage
+    resources:
+      - kadalustorages
+    verbs:
+      - watch
+---
+# CSI External Attacher
+# https://github.com/kubernetes-csi/external-attacher/blob/master/deploy/kubernetes/rbac.yaml
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: kadalu-csi-external-attacher
+rules:
+  - apiGroups: [""]
+    resources: ["persistentvolumes"]
+    verbs: ["get", "list", "watch", "patch"]
+  - apiGroups: ["storage.k8s.io"]
+    resources: ["csinodes"]
+    verbs: ["get", "list", "watch"]
+  - apiGroups: ["storage.k8s.io"]
+    resources: ["volumeattachments"]
+    verbs: ["get", "list", "watch", "patch"]
+  - apiGroups: ["storage.k8s.io"]
+    resources: ["volumeattachments/status"]
+    verbs: ["patch"]
```

Comment:
- kadalu doesn't use attacher sidecar and same will be informed to orchestrator via csidriver obj
- pls remove these and I believe these weren't there in current yaml files as well 🤔

Reply:
I thought it was used since: https://github.com/kadalu/kadalu/blob/devel/templates/csi.yaml.j2#L224
@leelavg  Please recheck, otherwise I can remove it of course.

Reply:
- Good find and I'm sure we don't need this
- Corresponding sidecar can also be removed 😀

Reply:
Ok, I'll remove them from here. Opened a bug report regarding running sidecar: https://github.com/kadalu/kadalu/issues/861